tests/test_flow/test_persistence_fuzz.py

```diff
@@ -24,10 +24,10 @@ def __init__(self, builder):
         self._entities_by_name = {}
         self._last_called_names = []
 
-    def add_entity(self, dep_names):
+    def add_entity(self, dep_names, changes_per_run=False):
```

Comment:
Yeah I can call in nondeterministic everywhere. It was hard to use charges per run, partly because it was awkward sometimes and partly because the names were getting huge.